.nvmrc

```diff
@@ -0,0 +1 @@
+18.16.0
```

Comment:
Just a question about the `.nvmrc` file: do we really need this? Isn't the `.node-version` file enough?